app/Widgets/Hero_Slider_Widget.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Boilerplate\Theme\Widgets;
+
+use JustCoded\WP\Framework\Page_Builder\v25\Page_Builder_Widget;
+use JustCoded\WP\Framework\Web\View;
+
+/**
+ * Class Hero_Slider_Widget
+ * Custom image wordpress widget based on siteorigin widgets bundle
+ */
+class Hero_Slider_Widget extends Page_Builder_Widget {
+
+	/**
+	 * Default slider type
+	 */
+	const TYPE_FULL = 'hero-slider-full';
```

Comment:
you have 2 options, so you should have 2 constants

Reply:
this constant was not found, then I created this one

Reply:
created